src/components/RequestInfo.tsx

```diff
@@ -0,0 +1,317 @@
+                                                captureCustomEventWithPageData('contact_us_submit', undefined, true)
+                                            }}
+                                            customSubmitButton='
+                                            <div class="button-container">
+                                            <button type="submit" class="custom-submit-button">
+                                            <div class="custom-submit-button-div">
+                                                <div className="!text-white">Request your demo</div>
+                                                <img src="/assets/send.svg" />
+                                            </div>
+                                            </button>
+                                            </div>'
+                                        />
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </ContentSection>
+            }
+        >
+            <ContentSection parentClassName="!my-0 !py-0 !pb-8 lg:hidden" className="!my-0 !py-0 ">
+                <div className="flex flex-col lg:hidden">
+                    <SocialProofSection iconItems={SocialProofIcons} />
+                    <CaseStudySection />
+                </div>
+            </ContentSection>
+
+            <ExpectationsSection items={expectationItems} />
+            <LinksCardSection />
+            <div className="sg-demo-gradient h-44 w-full" />
+        </Layout>
+    )
+}
+
+interface HeaderListTabProps {
+    contents: { icon: ReactNode; header: string; listItems: string[] }[]
+}
+
+const HeaderListTab: FunctionComponent<HeaderListTabProps> = ({ contents }) => (
+    <div className="flex flex-col gap-10">
+        {contents.map(content => (
+            <div key={content.header} className="flex flex-row gap-x-4">
+                <div>{content.icon}</div>
+                <div className="flex flex-col gap-y-2">
+                    <div className="text-xl font-semibold leading-[26px] tracking-tight text-gray-700">
+                        {content.header}
+                    </div>
+                    <ul className="ml-0 list-inside pl-3 text-lg leading-[27px] text-gray-500">
+                        {content.listItems.map(listItem => (
+                            <li key={listItem}>{listItem}</li>
+                        ))}
+                    </ul>
+                </div>
+            </div>
+        ))}
+    </div>
+)
+
+interface SocialProofIconsProps {
+    iconItems: {
+        id: string
+        src: string
+        className: string
+    }[]
+}
+
+const SocialProofSection: FunctionComponent<SocialProofIconsProps> = ({ iconItems }) => (
+    <div>
+        <div className="font-mono text-lg leading-[27px] tracking-normal text-[#5E6E8C]">
```

Comment:
Can we have this heading centered on mobile? 
<img width="803" alt="Screenshot 2024-10-14 at 11 16 08 AM" src="https://github.com/user-attachments/assets/d39df2c2-7ec8-4796-8fc2-262fd97418b0">